tests/unit-tests/newsletters.php

```diff
@@ -1,99 +0,0 @@
-<?php
-/**
- * Tests the Newsletters integration.
- *
- * @package Newspack\Tests
- */
-
-use Newspack\Newspack_Newsletters;
-use Newspack\Reader_Activation;
-
-/**
- * Tests the Newsletters integration.
- */
-class Newspack_Test_Newspack_Newsletters extends WP_UnitTestCase {
-	/**
-	 * Test the "master list" feature.
-	 */
-	public function test_newsletters_master_list_feature() {
```

Comment:
Removed because the `add_activecampaign_master_list` method is being deprecated in this PR.